prepline_sec_filings/api/section.py

```diff
@@ -66,6 +106,11 @@ def pipeline_api(text, m_section=[]):
         results[section] = sec_document.get_section_narrative(
             section_string_to_enum[section]
         )
+    for i, section_regex in enumerate(m_section_regex):
+        regex_enum = get_regex_enum(section_regex)
+        with timeout(seconds=60):
```

Comment:
For a single regex call on a single narrative element, isn't anything over 1 second already pretty much egregious?

Reply:
Yeah I can chop that down. I'll make it 5 seconds

Reply:
For the record, my earlier comment "single regex call on a single narrative element" also isn't true, it's potentially for all elements. Still, 5 seconds sounds good.